R/safe2x2Test.R

```diff
@@ -35,6 +35,10 @@
 #' @param previousSafeTestResult optionally, a previous safe test result can be provided. The posterior
 #' of the hyperparameters of this test is then used for the hyperparameter settings. Default NULL.
 #' @param M number of simulations used to estimate power or nBlocksPlan. Default \code{1000}.
+#' @param simThetaAMin minimal event rate in control group to simulate NPlan or power for.
+#' Can be specified when specifically interested in planning studies for specific event rates.
+#' Default \code{NULL}, then the entire parameter space (possibly restricted by delta) is used for simulation.
+#' @param simThetaAMax maximal event rate in control group to simulate NPlan or power for. Default \code{NULL}.
 #'
 #' @return Returns a 'safeDesign' object that includes:
 #'
```

Comment:
Perhaps add an example when the nuisance parameter is known

Reply:
Added the example